hazelcast-documentation/src/ThreadingModel.md

```diff
@@ -13,6 +13,8 @@ Hazelcast uses a pool of threads for I/O. A single thread does not do all the IO
 You can configure the number of IO-threads using the `hazelcast.io.thread.count` system property. Its default value is 3 per member. 
 This means that if 3 is used, in total there are 7 IO-threads; 1 accept-IO-thread, 3 read-IO-threads, and 3 write-IO-threads. Each IO-thread has its own Selector instance and waits on `Selector.select` if there is nothing to do.
 
+Hazelcast periodically scans utilization of each IO-thread and can decide to migrate a connection to a new thread if the existing thread is servicing disproportionate number of IO events. You can customize the scanning interval by setting a System Property `hazelcast.io.balancer.interval.seconds`. The default interval is 20 seconds. You can disable the balancing process by setting the property to a negative value.          
+
```

Comment:
Can you add a bit of comment what the purpose is of the feature? I would make that the initial line of the paragraph.